pkg/sysfs/syspath_test.go

```diff
@@ -596,7 +647,8 @@ func TestSysFsDeviceGetDeviceType(t *testing.T) {
 			sysfsDevice: &Device{
 				deviceName: "sdb1",
 				path:       "/dev/sdb1",
-				sysPath:    "/tmp/sys/devices/pci0000:00/0000:00:1f.2/ata1/host0/target0:0:0/0:0:0:0/block/sdb/sdb1/",
+				sysPath: filepath.Join(tmpDir,
+					"sys/devices/pci0000:00/0000:00:1f.2/ata1/host0/target0:0:0/0:0:0:0/block/sdb/sdb1/") + "/",
```

Comment:
```suggestion
					"sys/devices/pci0000:00/0000:00:1f.2/ata1/host0/target0:0:0/0:0:0:0/block/sdb/sdb1") + "/",
```

Reply:
If using `filepath.Join()`, I dont think its necessary to mention that `/` at the end.

Reply:
you mean the `+ "/"` part?

Reply:
nope. The `/` at the end of `block/sdb/sdb1/`